modules/tinymce/src/core/main/ts/api/html/Schema.ts

```diff
@@ -558,6 +566,12 @@ const Schema = (settings: SchemaSettings = {}): Schema => {
     return false;
   };
 
+  const isBlock = (name: string): boolean => Obj.has(getBlockElements(), name);
+
+  const isInline = (name: string): boolean => isValid(name) && !isBlock(name);
+
+  const isWrapElement = (name: string): boolean => Obj.has(getWrapBlockElements(), name) || isInline(name);
```

Comment:
I think we name this as `isWrapper` since the `isBlock` and `isInline` is not called `isBlockElement` also it takes a string name not a element. Maybe later we add predicates that are `isBlockElement` then this one would conflict with that. It's a bit weird predicate it only kind of affects copy/paste but I have no better name for it. Maybe `isContextWrapper` or something since it has to do with the context being added if you copy from `<h1>f[o]o</h1>` and that would result in `<h1>o</h1>`.

- [ ] Change name so that it wouldn't conflict if we add node predicates in the future

Reply:
ok, changed in `isWrapper`